pages/tissue-summary/[taxa].tsx

```diff
@@ -73,6 +82,18 @@ const taxaQueries: Record<string, TaxaMap> = {
     dataCellClass: "bg-tissue-matrix-data-mus-musculus",
     hoverCellClass: "group-hover:bg-tissue-matrix-highlight-mus-musculus",
   },
+  "homo-sapiens": {
+    dataProviderPath:
+      "/tissue-homo-sapiens/?type=Tissue&taxa=Homo+sapiens&virtual=false",
+    cellLinkPath: "/search/?type=Tissue&taxa=Homo+sapiens",
+    pagePath: "/tissue-summary/homo-sapiens",
+    queryAtType: "TissueSummaryHomoSapiens",
+    columnHeaderCellClass: "bg-tissue-matrix-column-header-mus-musculus",
```

Comment:
do you use the same class for human and mouse? Or you have different class for human?

Reply:
Oh, I _do_ have different ones already defined for human that I forgot about. They use exactly the same colors (for now), but I’ll change these classes.

Reply:
Everything else is fine except this cosmic issue. I just approved the PR.